runtime/parser2/declaration.go

```diff
@@ -562,6 +570,31 @@ func parseImportDeclaration(p *parser) *ast.ImportDeclaration {
 	}
 }
 
+// isCurrentTokenAnImportedIdentifier check whether the current token
+// needs to be processed as an imported identifier or not.
+func isCurrentTokenAnImportedIdentifier(p *parser) bool {
```

Comment:
Given that this function is considering the next token (it skips the current), maybe rename this function